course/src/main/java/org/openedx/course/presentation/CourseRouter.kt

```diff
@@ -24,22 +24,25 @@ interface CourseRouter {
     fun navigateToCourseSubsections(
         fm: FragmentManager,
         courseId: String,
-        blockId: String,
-        mode: CourseViewMode,
-        descendantId: String? = ""
+        subSectionId: String,
+        unitId: String? = "",
```

Comment:
Not very clear why there is nullable argument with the default empty string value.

Reply:
removed `nullable`